bip-witaddr.mediawiki

```diff
@@ -19,7 +19,7 @@ This document proposes a checksummed base32 format, "Bech32", and a standard for
 
 ===Copyright===
 
-This BIP is licensed under the 2-clause BSD license.
+This BIP is licensed under the [https://github.com/bitcoin/bips/blob/master/COPYING 2-clause BSD license].
```

Comment:
I don't think this is common practice in BIPs, as after merging into the bips repository, it will become a single file.